src/helpers/print.js

```diff
@@ -10,14 +10,14 @@ import core from 'core';
 
 let pendingCanvases = [];
 let includeAnnotations = false;
-let printQuality = 1;
+let PRINT_QUALITY = 1;
 let colorMap;
 
 export const print = async(dispatch, isEmbedPrintSupported, sortStrategy, colorMap, options) => {
-  let includeAnnotations, includeComments, pagesToPrint, onProgress;
+  let includeAnnotations, includeComments, pagesToPrint, onProgress, printQuality;
   let printWithoutModal = false;
   if (options) {
-    ({ includeAnnotations, includeComments, pagesToPrint, onProgress, printWithoutModal } = options);
+    ({ includeAnnotations, includeComments, pagesToPrint, onProgress, printWithoutModal, printQuality } = options);
```

Comment:
Drop if condition and rather have default assignment in method signature, as in `async(dispatch, isEmbedPrintSupported, sortStrategy, colorMap, options = {}) => {…}`